tests/e2e/support/objects/app-files/resource/index.ts

```diff
@@ -211,4 +211,8 @@ export class Resource {
   async clickTag(args: Omit<clickTagArgs, 'page'>): Promise<void> {
     return await clickResourceTag({ ...args, page: this.#page })
   }
+
+  async openFileFolder(resource): Promise<void> {
+    await clickResource({ page: this.#page, path: resource })
+  }
```

Comment:
Very similar to https://github.com/owncloud/web/blob/625bbdc3c12fad4232906ac563413c8b6c5af595/tests/e2e/support/objects/app-files/resource/index.ts#L183
```suggestion
```

Reply:
Thanx. Missed refactoring that.